src/promptflow/tests/executor/unittests/_utils/test_exception_utils.py

```diff
@@ -229,6 +229,29 @@ def test_to_dict_for_tool_execution_error(self):
             },
         }
 
+    def test_error_codes_for_general_exception(self):
```

Comment:
I think we can merge this method with the below one and use the `pytest.mark.parametrize` mark.

Reply:
Done.